apps/backend/src/handlers/getAllForms.ts

```diff
@@ -1,25 +1,36 @@
 import { APIGatewayEvent } from 'aws-lambda';
+import { createTableIfNotExists } from '../db/createTable.js';
 import { fetchDocuments } from '../db/utils.js';
-import { executeHandler } from '../utils/executeHandler.js';
-import { validateMethodType } from '../utils/validateMethodType.js';
+import { createResponse } from '../utils/createResponse.js';
 
 /**
  * An HTTP get method to get all forms from the QLDB table.
  */
 export const getAllFormsHandler = async (event: APIGatewayEvent) => {
-  let response = validateMethodType(
-    event.httpMethod,
-    'GET',
-    'getAllFormsHandler'
-  );
-  if (response) {
-    return response;
+  if (event.httpMethod !== 'GET') {
+    return createResponse(
+      400,
```

Comment:
can you reuse Nate's refactored helper methods? e.g. `validateMethodType` instead of `if (event.httpMethod !== 'GET')`

Reply:
also use `executeHandler`, take a look at what main looks like right now for how to use Nate's helpers